test/integration/file-uploads/request-body.test.js

```diff
@@ -117,11 +117,10 @@ describe('file upload in request body', function () {
 
             var resp = JSON.parse(testrun.response.getCall(3).args[2].stream.toString());
 
-            expect(resp).to.have.property('data', '{\n\t"key1":"value1",\n\t"key2": 2\n}\n');
+            expect(resp.data).to.be.eql({key1: 'value1', key2: 2});
             expect(resp).to.nested.include({
                 'headers.content-length': '33'
             });
-            expect(resp.headers['content-type']).to.equal('text/plain');
```

Comment:
Don't remove this assertion. Instead, add an assertion for missing/inferred `Content-Type` from the file type.

Reply:
We have content-type as  '' (empty string). Should I add expect for empty string?